assets/js/components/PresenterStreamArea.tsx

```diff
@@ -12,12 +14,59 @@ type PresenterStreamAreaProps = {
 export const presenterStreams: { [key: string]: MediaStream } = {};
 let webrtc: MembraneWebRTC | null = null;
 
+export const changeIsTrackEnabled = (clientName: string, sourceType: SourceType, mode: boolean) => {
+  const track = presenterStreams[clientName].getTracks().find((track) => track.kind == sourceType);
+  if (track != undefined) {
+    track.enabled = mode;
+  }
+};
+
+const addMediaStreamTrack = (
+  clientName: string,
+  mediaStream: MediaStream,
+  sourceType: SourceType
+) => {
+  if (presenterStreams[clientName] == undefined) presenterStreams[clientName] = new MediaStream();
+  else {
+    const curTrack = presenterStreams[clientName]
+      .getTracks()
+      .find((track) => track.kind == sourceType);
+    if (curTrack != undefined) {
+      curTrack.stop();
+      presenterStreams[clientName].removeTrack(curTrack);
+    }
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/API/MediaStream nothing happens if the passed track doesn't exist in the media stream.
```suggestion
      curTrack?.stop();
      presenterStreams[clientName].removeTrack(curTrack);
    }
```

Reply:
But that's not the case here, argument passed to removeTrack must not be undefined (or am I missing something?)

Reply:
Done